example/src/CSE.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Button, SafeAreaView } from 'react-native';
+import { NativeModules } from 'react-native';
+import { ENVIRONMENT } from './Configuration';
+import Styles from './Styles';
+
+const { AdyenCSE } = NativeModules;
+
+const CseView = () => {
+  const tryEncryptCard = async () => {
+    let unencryptedCard = {
```

Comment:
Should be defined as `const` instead of `let`, as there is no need to mutate the object

Reply:
This will probably change (or maybe even removed).
I'll address this in next PR